cpp/oneapi/dal/algo/dbscan/test/batch.cpp

```diff
@@ -30,14 +30,19 @@ namespace oneapi::dal::dbscan::test {
 namespace te = dal::test::engine;
 namespace la = te::linalg;
 
+constexpr inline std::uint64_t mask_full = 0xffffffffffffffff;
+
```

Comment:
What for?

Reply:
It is used at line 159 to check how it works with all bits set